tests/clients/api/test_datafeed_client_v2.py

```diff
@@ -1,133 +1,136 @@
-import pytest
-from unittest.mock import patch
 import json
 import os
+import unittest
+from unittest.mock import patch
 
-from sym_api_client_python.clients.sym_bot_client import SymBotClient
 from sym_api_client_python.auth.rsa_auth import SymBotRSAAuth
+from sym_api_client_python.clients.sym_bot_client import SymBotClient
 from sym_api_client_python.configure.configure import SymConfig
 
 
-@pytest.fixture
-def datafeed_client():
-    configure = SymConfig('./tests/resources/bot-config.json', __file__)
-    configure.load_config()
-    configure.data["datafeedVersion"] = "v2"
-
-    # auth = SymBotRSAAuth(configure)
-    auth = SymBotRSAAuth(configure)
-    auth.session_token = "eyJhbGciOiJSUzUxMiJ9.eyJzdWIiOiJ0ZXN0LWJvdCIsImlzcyI6InN5bXBob255Iiwic2Vzc2lvbklkIjoiZmRiOTAxMmQzOTgwMGE3NzNkMTJjYWFmZGY5MjU4ZjZjOWEyMTE2MmYyZDU1ODQ3M2Y5ZDU5MDUyNjA0Mjg1ZjU0MWM5Yzg0Mzc5YTE0MjZmODNiZmZkZTljYmQ5NjRjMDAwMDAxNmRmMjMyODIwNTAwMDEzZmYwMDAwMDAxZTgiLCJ1c2VySWQiOiIzNTE3NzUwMDE0MTIwNzIifQ.DlQ_-sAqZLlAcVTr7t_PaYt_Muq_P82yYrtbEEZWMpHMl-7qCciwfi3uXns7oRbc1uvOrhQd603VKQJzQxaZBZBVlUPS-2ysH0tBpCS57ocTS6ZwtQwPLCZYdT-EZ70EzQ95kG6P5TrLENH6UveohgeDdmyzSPOEiwyEUjjmzaXFE8Tu0R3xQDwl-BKbsyUAAgd1X7T0cUDC3WIDl9xaTvyxavep4ZJnZJl4qPc1Tan0yU7JrxtXeD8uwNYlKLudT3UVxduFPMQP_2jyj5Laa-YWGKvRtXkcy2d3hzf4ll1l1wVnyJc1e6hW2EnRlff_Nxge-QCJMcZ_ALrpOUtAyQ"
+class TestDataFeedClientV2(unittest.TestCase):
```

Comment:
You can add this line above the class: `@patch('sym_api_client_python.clients.sym_bot_client.requests.sessions.Session.request')`

Then add `mock_request` to the argument of all test function, ex: `def test_create_datafeed(self, mock_request):`
By doing so, you can remove all the `with` statement inside testcase to avoid duplicating code.